### ISSUE TITLE
Post has been corrected. Close the issue.Deleted items in Configure still appear in Apps Launcher

### ISSUE THREAD

**Issue by ckbeme** [gregolsky/chrome-app-launcher#42]
gregolsky - I'm very  sorry about this post. I hope you haven't begun any troubleshooting yet.
I shutdown chrome, then went into chrome on my other PC. The Apps were all correct. So I'm guessing you have to close chrome for the changes in App Launcher to take effect.
So....  Solved - never was a problem.

Thanks again for a very 'nifty' add on for Chrome.

I looked through the issues, and the only similar issue: Someone said all of their items were not showing up.
When I added your extension, I assumed it was going to configure items listed on the "Apps" multicolored button on the Chrome toolbar. I don't know it's supposed to work this way, but I have to access the App Launcher when I'm in my Gmail Account.

![google app launcher from gmail](https://cloud.githubusercontent.com/assets/17622963/15368026/6c97d12a-1cf1-11e6-921d-8b1f563ea08a.jpg)

![part 1 of configured apps](https://cloud.githubusercontent.com/assets/17622963/15368053/8b66f162-1cf1-11e6-8fde-1e7256124689.jpg)
![part 2 of configured apps](https://cloud.githubusercontent.com/assets/17622963/15368058/9158891e-1cf1-11e6-8c09-e3e62333790a.jpg)

**Comment by ckbeme** [comment]
Pic 1 is list from my gmail screen. Pic 2 is the first part of my configured apps from your extension.
Pic 3 is the rest of the configured apps. I deleted from you app - Voice, Swiffy, Apps, Vault, and that big yellow X (can't remember what it is)

Thanks for any help. I love this extension, and want it to work for me. I may very well be doing something wrong.

Cathy

**Comment by ckbeme** [comment]
See my subject line and new paragraph I added at beginning of this post.

**Comment by gregolsky** [comment]
No worries. Just to confirm, I have no power over apps listed in the launcher available through Google services like Gmail. It's all Google. The only thing this extensions I am sure is in-sync with, is the chrome://apps list of Chrome apps.

**Comment by ckbeme** [comment]
Yesterday was the first time I'd been on Github, and in my post edit I said 'close the issue' cuz I didn't know I could do it.


I really appreciate your add on.  First thing I thought when I saw the Google Apps page was "Too bad u can't modify the content."

Thanks for a brilliant and much needed add on. And it's so simple to use. 


Regards,


Cathy 


Get Outlook for Android


From: Grzegorz Lachowski

Sent: Wednesday, May 18, 5:02 PM

Subject: Re: [gregolsky/chrome-app-launcher] Post has been corrected. Close the issue.Deleted items in Configure still appear in Apps Launcher (#42)

To: gregolsky/chrome-app-launcher

Cc: ckbeme, State change


No worries. Just to confirm, I have no power over apps listed in the launcher available through Google services like Gmail. It's all Google. The only thing this extensions I am sure is in-sync with, is the chrome://apps list of Chrome apps.
